src/main/java/com/google/edith/App.java

```diff
@@ -1,13 +1,8 @@
 package com.google.edith;
 
-/**
- * Hello world!
- *
- */
-public class App 
-{
-    public static void main( String[] args )
-    {
-        System.out.println( "Hello World!" );
-    }
+/** Hello world! */
+public class App {
+  public static void main(String[] args) {
+    System.out.println("Hello World!");
```

Comment:
This doesn't seem to do anything important. Is it necessary?

Reply:
Yeah, it does not do anything. It was just boilerplate code generated while bootstrapping the app. Should I delete it ?

Reply:
Yeah, remove it if you can.

Reply:
Removed it. 
Changes made.